docs/user-guide/templates/pipeline-templates.md

```diff
@@ -85,6 +88,90 @@ If no template version is specified, the most recently published will be used. T
 
 The most reliable way to avoid unexpected template changes is to refer to a specific version of the template. For instance, `nodejs/test@1.0.4` is an immutable reference to a particular list of steps. Using a reference such as `nodejs/test@1.0` means that a job will automatically use `nodejs/test@1.0.5` when it becomes available, but that comes with risk of an unexpected change in behavior.
 
+### Customization
+Some customization can be done within the `jobs` configuration when using a Pipeline Template.
+
+For user-defined jobs with names that *already exist* in the pipeline template, customization is limited to a set of certain fields: `image`, `settings`, `environment`, and `requires`.
+
+For user-defined jobs with names that *do not already exist* in the pipeline template, all jobs fields are allowed.
+
+#### Example
+Given a pipeline template:
+
+```
+shared:
+  image: node:10
```

Comment:
nitpick: Let's use node version 18 or above, in the example, some users may just copy-paste the configuration and end up having an error.